subsys/net/ip/Kconfig.mgmt

```diff
@@ -95,6 +95,14 @@ config NET_MGMT_EVENT_INFO
 	  and listeners will then be able to get it. Such information depends
 	  on the type of event.
 
+if NET_MGMT_EVENT_INFO
+config NET_MGMT_EVENT_INFO_DEFAULT_DATA_SIZE
+	int "Default size of event information data"
+	default 32
+	help
+	  The default size of the data which can be passed along with an event.
+endif
```

Comment:
extend the if to after `config NET_MGMT_EVENT_MONITOR` so `NET_MGMT_EVENT_INFO` can be removed from its depends on